docs/community/local-groups.mdx

```diff
@@ -17,6 +17,8 @@ organizer with an online presence and wish to be included in this list, please e
 us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 
 ## Australia
+### Australian Capital Territory
+- [Meshtastic Canberra - Discord](https://discord.gg/4QgFsuaC3Z)
```

Comment:
Thanks for being part of the Meshtastic community. Would it be possible to change the name of the group both on discord as well as in this pull request to align with https://meshtastic.org/docs/legal/licensing-and-trademark/#noncommercial-and-community-web-sites ? A suggested update would be to add a 'User Group', 'Club' or 'Users' denominator, so to not be confused with a official meshtastic site.